gnomad/assessment/validity_checks.py

```diff
@@ -546,10 +549,25 @@ def summarize_variants(
         var_summary.contigs,
     )
 
+    # Check that all contigs have variant calls.
     for contig in var_summary.contigs:
         if var_summary.contigs[contig] == 0:
             logger.warning("%s has no variants called", var_summary.contigs)
 
+    # Check that all expected contigs are found in the variant summary
+    # and that no unexpected contigs are found.
+    if expected_contigs:
+        var_summary_contigs = var_summary["contigs"].keys()
+        missing_contigs = expected_contigs - var_summary_contigs
+        unexpected_contigs = var_summary_contigs - expected_contigs
+
+        if missing_contigs or unexpected_contigs:
+            logger.warning(
+                "FAILED contig check. Expected contigs and found contigs do not match! Expected: %s, Found: %s",
+                expected_contigs,
+                var_summary_contigs,
```

Comment:
```suggestion
                list(var_summary_contigs),
```
very small note, but the error reporting this prints var_summary_contigs as a dict.keys(['chr1','chr11'...]) made me thing there was a type error at first. All of the actual code and logic for this is fine, but printing the list() version to make it the exact same type as expected_contigs could be clearer